odxtools/diaglayers/ecuvariant.py

```diff
@@ -38,6 +39,14 @@ def dyn_defined_spec(self) -> Optional[DynDefinedSpec]:
     def parent_refs(self) -> List[ParentRef]:
         return self.ecu_variant_raw.parent_refs
 
+    @property
+    def base_variant(self) -> Optional[BaseVariant]:
+        for pr in self.ecu_variant_raw.parent_refs:
+            if isinstance(pr.layer, BaseVariant):
```

Comment:
Could be multiple or not?

Reply:
the spec only at most a single base variant parent, see checker rule 50 at the bottom. I've added a docstring. (sorry for the "drive-by change", BTW.)